src/test/python/test_scc_time_codes.py

```diff
@@ -51,32 +52,227 @@ def test_parse_drop_frame_time_code(self):
     self.assertEqual(3, time_code.get_seconds())
     self.assertEqual(4, time_code.get_frames())
     self.assertTrue(time_code.is_drop_frame())
-    self.assertEqual(111582, int(time_code._get_frames()))
-    self.assertAlmostEqual(3719.414, float(time_code.get_fraction()), delta=0.001)
+    self.assertEqual(111582, time_code.get_nb_frames())
+    self.assertAlmostEqual(111582 / 30, float(time_code.get_fraction()), delta=0.001)
```

Comment:
shouldn't this be:

```self.assertEqual(111582 / Fraction(30000/1001), time_code.get_fraction())```

Reply:
No, in my opinion it wouldn't make sense since the number of frames (used as numerator) is absolute. That's why the fraction is computed with the non-drop-frame frame rate as denominator by default.

Reply:
Fixed in https://github.com/sandflow/ttconv/pull/46/commits/2580698102641c728205ac27b125fdb157849b34